package.json

```diff
@@ -66,5 +67,6 @@
   "dependencies": {
     "debug": "^4.2.0",
     "follow-redirects": "^1.13.0"
-  }
+  },
+  "babel": {}
```

Comment:
Is this required?

Reply:
yeah, latest version of eslint-wpvip-config requires @babel/eslint-parser, which fails without a babel config (in repos that don't use babel). This fixes the problem